index.html

```diff
@@ -274,6 +275,43 @@ <h1 class="underlined">About Me</h1>
                 </ul>
             </div>
         </section>
+
+        <!-- contact me  -->
+        <section class="get-in-touch">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form action="https://formspree.io/f/xyyawvzk" enctype="multipart/form-data" method="POST"
+                class="get-in-touch-form">
+                <div class="name-input">
+                    <input type="text" name="first name" placeholder="First Name" maxlength="30" required>
+                    <input class="last-name-input" type="text" name="last name" placeholder="Last Name" maxlength="30"
+                        required>
+                </div><input type="email" name="email" placeholder="Email Address" required>
+                <textarea name="message" id="message" cols="30" rows="5" maxlength="500" required
+                    placeholder="Enter text here">
+                </textarea>
```

Comment:
- [ ] **[OPTIONAL]** Please consider removing the blank line and spaces inside the `<textarea>`. It's not highly visible, but there is a blank space on the texteara between lines 292 and 293.
**your code should look like this without any blank lines:**
```
                <textarea name="message" id="message" cols="30" rows="5" maxlength="500" required
                    placeholder="Enter text here"></textarea>
```
This will make sure the placeholder appears correctly when the website just loaded.
**Currently the placeholder is not visible before removing the blank line**
![image](https://user-images.githubusercontent.com/74411135/228924536-76da701a-6036-49ca-a2a3-b54550ebe664.png)

This will improve the user experience 📱 and make your code look way cleaner 🧼  and professional 😄

Reply:
Resolved Required Changes

- Removed the blank line and spaces inside the <textarea>.

- Button interaction like hover added

- Already contact form submission is working perfectly when you fill out all forms and then click on submit button it works perfectly.

- Removed commented line of code